downstream/modules/platform/proc-operator-external-db-hub.adoc

```diff
@@ -22,7 +22,7 @@ The external database must be a PostgreSQL database that is the version supporte
 
 [NOTE]
 ====
-{PlatformNameShort} {PlatformVers} supports PostgreSQL 13.
+{PlatformNameShort} {PlatformVers} supports PostgreSQL 15.
```

Comment:
PostgreSQL 15 could be replaced with `{PostgresVers}`

Reply:
Done!